### PR TITLE
ISSDEV-2045 Samples are updated with tanks as supported hardware

### PR DESCRIPTION
**Pull request by andrey4latyshev**
Initial commit for the issue 2045.
The question is what should be added in README.md files? Just one more board (Tank) as supported and tested?

### CODE REVIEW

File: interrupt/maven/src/main/java/iotdk/example/Interrupt.java [hunk 1]
```diff
@@ -95,6 +95,8 @@ public static void initPlatform(){
             pinNumber = 26;
         if(platform.equals(Platform.INTEL_JOULE_EXPANSION))
             pinNumber = 26;
+        if (platform.equals(Platform.IEI_TANK))_
```
Comment (sasmita) [inline interrupt/maven/src/main/java/iotdk/example/Interrupt.java]: What is the use of underscore here? Was this intentional? 
Everything else LGTM.

Comment (andrey4latyshev) [reply interrupt/maven/src/main/java/iotdk/example/Interrupt.java]: > What is the use of underscore here? Was this intentional?
> Everything else LGTM.

No, I pressed it unintentionally. Just fixed it. Thanks, Sasmita!

File: digital-in/cpp/digital-in.cpp [hunk 2]
```diff
@@ -77,6 +77,9 @@ void initPlatform(int& gpioPin)
     case INTEL_JOULE_EXPANSION:
         gpioPin = 101;
         break;
+    case IEI_TANK:
+    	gpioPin = 1;
```
Comment (Propanu) [inline digital-in/cpp/digital-in.cpp]: Use an input pin here, pin 2.

File: digital-in/maven/src/main/java/iotdk/example/DigitalIn.java [hunk 3]
```diff
@@ -83,6 +83,8 @@ public static void initPlatform()
                 mraa.addSubplatform(Platform.GROVEPI, "0");
                 pinNumber = 4 + 512; // D4 Connector (512 offset needed for the shield)
             }
+        } else if (platform.equals(Platform.IEI_TANK)) {
+        	pinNumber = 1;
```
Comment (Propanu) [inline digital-in/maven/src/main/java/iotdk/example/DigitalIn.java]: Use an input pin here, pin 2.

File: digital-out/cpp/digital-out.cpp [hunk 4]
```diff
@@ -77,6 +77,9 @@ void initPlatform(int& gpioPin)
     case INTEL_JOULE_EXPANSION:
         gpioPin = 101;
         break;
+    case IEI_TANK:
+    	gpioPin = 2;
```
Comment (Propanu) [inline digital-out/cpp/digital-out.cpp]: Use an output pin here, pin 1.

File: digital-out/maven/src/main/java/iotdk/example/DigitalOut.java [hunk 5]
```diff
@@ -78,6 +78,8 @@ public static void initPlatform(){
                 mraa.addSubplatform(Platform.GROVEPI, "0");
                 pinNumber = 4 + 512; // D4 Connector (512 offset needed for the shield)
             }
+        } else if (platform.equals(Platform.IEI_TANK)) {
+        	pinNumber = 2;
```
Comment (Propanu) [inline digital-out/maven/src/main/java/iotdk/example/DigitalOut.java]: Use an output pin here, pin 1.

File: interrupt/cpp/interrupt.cpp [hunk 6]
```diff
@@ -83,6 +83,9 @@ void initPlatform(int& gpioPin)
     case INTEL_JOULE_EXPANSION:
         gpioPin = 26;
         break;
+    case IEI_TANK:
+    	gpioPin = 1;
```
Comment (Propanu) [inline interrupt/cpp/interrupt.cpp]: ISRs attach to inputs, pin 2.

File: interrupt/maven/src/main/java/iotdk/example/Interrupt.java [hunk 7]
```diff
@@ -95,6 +95,8 @@ public static void initPlatform(){
             pinNumber = 26;
         if(platform.equals(Platform.INTEL_JOULE_EXPANSION))
             pinNumber = 26;
+        if (platform.equals(Platform.IEI_TANK))
```
Comment (Propanu) [inline interrupt/maven/src/main/java/iotdk/example/Interrupt.java]: ISRs attach to inputs, pin 2.

File: onboard-blink/cpp/blink.cpp [hunk 8]
```diff
@@ -78,6 +78,9 @@ void initPlatform(int& gpioPin)
     case INTEL_JOULE_EXPANSION:
         gpioPin = 101;
         break;
+    case IEI_TANK:
+    	gpioPin = 2;
```
Comment (Propanu) [inline onboard-blink/cpp/blink.cpp]: Use an output pin here, pin 1.

**Comment by Propanu** [comment]
Also, in the readme files for the updated samples, under **Supported Boards** add the following:
* IEI* TANK AIoT Dev Kit

**Pull request closed without merging**